kythe/typescript/testdata/class.ts

```diff
@@ -33,11 +30,12 @@ class Class implements IFace {
     //- StaticMember.tag/static _
     static staticMember: number;
 
-    // TODO: ClassCtor should really point at this constructor, not at the
-    // top-level class declaration.
     // This ctor declares a new member var named 'otherMember', and also
     // declares an ordinary parameter named 'member' (to ensure we don't get
     // confused about params to the ctor vs true member variables).
+    //- @constructor defines/binding ClassCtor
+    //- ClassCtor.node/kind function
+    //- ClassCtor.subkind constructor
     //- @otherMember defines/binding OtherMember
     //- OtherMember.node/kind variable
     //- @member defines/binding FakeMember
```

Comment:
Wondering if we capture the visibility on member vars? And if so, can you add a test for it here?

Reply:
Can you also test the scoping (childof) of both `member` and `otherMember`

Reply:
> Wondering if we capture the visibility on member vars? And if so, can you add a test for it here?

What do you mean by visibility?

Reply:
https://github.com/kythe/kythe/blob/master/kythe/data/schema_index.textproto#L112

defined in #2978 

but I don't think any indexers use it yet. So nevermind.